dotnetv3/Cognito/Scenarios/Cognito_Basics/UIMethods.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[Cognito.dotnetv3.UIMethods]
+namespace CognitoBasics;
+
+/// <summary>
+/// Some useful methods to make screen display easier.
+/// </summary>
+public class UiMethods
+{
+    public readonly string SepBar = new string('-', Console.WindowWidth);
+
+    /// <summary>
+    /// Show information about the scenario.
+    /// </summary>
+    public void DisplayOverview()
+    {
+        Console.Clear();
+        DisplayTitle("Welcome to the Amazon Cognito Demo");
+
+        Console.WriteLine("This example application will do the following:");
+        Console.WriteLine("\t 1. Sign up a user.");
+        Console.WriteLine("\t 2. Get the user's confirmation status.");
+        Console.WriteLine("\t 3. Resends the confirmation code if the user requested another code.");
+        Console.WriteLine("\t 4. Confirms that the user signed up.");
+        Console.WriteLine("\t 5. Invokes the initiateAuth to sign in. This results in being prompted to set up TOTP (time-based one-time password). (The response is “ChallengeName”: “MFA_SETUP”).");
+        Console.WriteLine("\t 6. Invokes the AssociateSoftwareToken method to generate a TOTP MFA private key. This can be used with Google Authenticator.");
+        Console.WriteLine("\t 7. Invokes the VerifySoftwareToken method to verify the TOTP and register for MFA.");
+        Console.WriteLine("\t 8. Invokes the AdminInitiateAuth to sign in again. This results in being prompted to submit a TOTP (Response: “ChallengeName”: “SOFTWARE_TOKEN_MFA”).");
+        Console.WriteLine("\t 9. Invokes the AdminRespondToAuthChallenge to get back a token.");
+    }
+
+    /// <summary>
+    /// Display a message and wait until the user presses enter.
+    /// </summary>
+    public void PressEnter()
+    {
+        Console.Write("\nPress <Enter> to continue.");
+        _ = Console.ReadLine();
+    }
+
+    /// <summary>
+    /// Pad a string with spaces to center it on the console display.
+    /// </summary>
+    /// <param name="strToCenter"></param>
+    /// <returns></returns>
```

Comment:
Are these lines missing some values, or intentionally blank?

Reply:
Not intentional. Fixed